src/widgets/ReportSideBar/ReportSideBar.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/no-unused-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, ListItem, FlatList, Text, StyleSheet } from 'react-native';
+import { APP_FONT_FAMILY, GREY } from '../../globalStyles';
+import { pageInfoStrings } from '../../localization/pageInfoStrings';
+
+export const ReportSideBar = ({ data, onPressItem, selectedItemIndex, dimensions }) => {
```

Comment:
Looks like this was merged into another branch, that's why it's coming up here https://github.com/openmsupply/mobile/pull/1763 if you were unsure!

Reply:
Yes, that happened because I created this branch from the other one and brang those files that then merged on feature/dashboard and never update this one. Thanks for noticing!